force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -68,7 +81,14 @@ public with sharing class VariablesDataTypesOperators {
      * @return The quotient of the division, or 0 if the denominator is zero.
      */
     public static Decimal division(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+        if (a == null || b == null) {
+            return null;
+        } else if (b == 0){
+            return 0;
+        }
+        else {
```

Comment:
Generally, put else and else if on the same line as the closed brackets. Spacing is not necessary.